src/components/Workflow/filters/useWorkflowShowArchivedState.ts

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { FilterOperation, FilterOperationName } from 'models/AdminEntity/types';
+import { WorkflowExecutionState } from 'models/Workflow/enums';
+
+interface ArchiveFilterState {
+  showArchived: boolean;
+  setShowArchived: (newValue: boolean) => void;
+  getFilter: () => FilterOperation;
+}
+
+/**
+ *  Allows to filter by Archive state
+ */
+export function useWorkflowShowArchivedState(): ArchiveFilterState {
+  const [showArchived, setShowArchived] = useState(false);
+
+  // By default all values are returned with NAMED_ENTITY_ACTIVE state,
+  // so filter need to be applied only for ARCHIVED executions
```

Comment:
Do we currently apply filter in both cases? If yes, than line 18 comment is not needed here.